packages/services/src/contents/index.ts

```diff
@@ -1080,8 +1115,21 @@ export class Drive implements Contents.IDrive {
     this._apiEndpoint = options.apiEndpoint ?? SERVICE_DRIVE_URL;
     this.serverSettings =
       options.serverSettings ?? ServerConnection.makeSettings();
+    const restContentProvider = new RestContentProvider({
+      apiEndpoint: this._apiEndpoint,
+      serverSettings: this.serverSettings,
+      fileChanged: this.fileChanged
+    });
+    this.contentProviderRegistry.register(restContentProvider);
```

Comment:
Instead of registering the rest API content provider as any other providers, should we set a default provider and use it as fallback to avoid filepath analysis when no other providers are defined?

For reference we are doing something like that in the document registry: https://github.com/jupyterlab/jupyterlab/blob/main/packages/docregistry/src/registry.ts#L41

Reply:
Done in ad6ae159c2535c05a274f19dc17463d8080f9c36